### PR TITLE
Update access request enterprise description

### PR DESCRIPTION
**Pull request by stevenGravy**
- Mention resource requests are ent only
- Mention UI for access requests are ent only

### CODE REVIEW

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx [hunk 1]
```diff
@@ -80,7 +80,9 @@ See our [FedRAMP Compliance for Infrastructure Access](../../access-controls/com
 
 ## Access Requests
 
-With Teleport we've introduced the ability for users to request additional roles. The Access Request API makes it easy to dynamically approve or deny these requests.
+With Teleport users can request access to specific resources and roles with requests publishable to tools such as Slack, Jira, PagerDuty that fits to your organization's workflows. Authorized users and automated processes can approve the requests. The commercial edition of Access Requests allows for
```
Comment (zmb3) [inline docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx:83]:
```suggestion
With Teleport, users can request access roles or to a specific set of resources.
Requests can be published to tools like Slack, Jira, and PagerDuty for easy integration with your organization's workflows.
```

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx [hunk 2]
```diff
@@ -80,7 +80,9 @@ See our [FedRAMP Compliance for Infrastructure Access](../../access-controls/com
...
+requesting specific resources such as a production server to allow fine-grain access for a targeted task.  The Web User Interface for Access Requests
```
Comment (zmb3) [inline docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx:84]: The web interface for creating and approving access requests is only available in the Teleport Enterprise.

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx [hunk 3]
```diff
@@ -15,7 +15,7 @@ The table below gives a quick overview of the benefits of Teleport Enterprise.
 | Teleport Enterprise Feature | Description |
 | - | - |
 | [Single Sign-On (SSO)](#sso) | Allows Teleport to integrate with existing enterprise identity systems. Examples include Active Directory, GitHub, Google Apps and numerous identity middleware solutions like Auth0, Okta, and so on. Teleport supports SAML and OAuth/OpenID Connect protocols to interact with them. |
-| [Access Requests](../../access-controls/access-requests.mdx) | User interface for teams to create and review requests to access infrastructure with escalated privileges. |
+| [Access Requests](../../access-controls/access-requests.mdx) | Request access to specific resources and a user interface for teams to create and review requests to access infrastructure with escalated privileges. |
```
Comment (zmb3) [inline docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx:18]:
```suggestion
| [Access Requests](../../access-controls/access-requests.mdx) | Request elevated access to roles or specific resources. |
```

**Comment by stevenGravy** [comment]
thanks for reviewing @zmb3 , updated!

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx [hunk 4]
```diff
@@ -80,7 +80,10 @@ See our [FedRAMP Compliance for Infrastructure Access](../../access-controls/com
 
 ## Access Requests
 
-With Teleport we've introduced the ability for users to request additional roles. The Access Request API makes it easy to dynamically approve or deny these requests.
+With Teleport, users can request access roles or to a specific set of resources.
+Requests can be published to tools like Slack, Jira, and PagerDuty for easy integration with your organization's workflows.
+requesting specific resources such as a production server to allow fine-grain access for a targeted task.  The web interface
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx:85]: > requesting specific resources such as a production server to allow fine-grain access for a targeted task.  

Looks like the first half of this sentence was cut off; there's no subject. Also, extra spacing at the end.

**Review by ptgott — APPROVED** [review]

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx (hunk 4 above)
Comment (stevenGravy) [reply docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx]: Please see updates

File: docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx [hunk 5]
```diff
@@ -80,7 +80,11 @@ See our [FedRAMP Compliance for Infrastructure Access](../../access-controls/com
 
 ## Access Requests
 
-With Teleport we've introduced the ability for users to request additional roles. The Access Request API makes it easy to dynamically approve or deny these requests.
+With Teleport, users can request access to roles or to a specific set of resources
+such as a production server. Requests can be published to tools like Slack, Jira,
+and PagerDuty for easy integration with your organization's workflows. The web 
+interface provides for creating and approving access requests that is only 
+available in Teleport Enterprise.
```
Comment (alexfornuto) [inline docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx:87]:
```suggestion
and PagerDuty for easy integration with your organization's workflows. Teleport
Enterprise's web interface also provides for creating and approving access requests.
```

**Review by alexfornuto — APPROVED** [review]

**Pull request merged**